### PR TITLE
adding choice component theme

### PR DESCRIPTION
**Pull request by forrestbaer**
## Summary

Added choice + choicelist component theming variables, added another spacer type

## How to test

`yarn loki test` `yarn loki:healthcare test` `yarn loki:medicare test`

### CODE REVIEW

File: packages/design-system-tokens/src/themes/core/defaultTheme.ts [hunk 1]
```diff
@@ -131,6 +135,29 @@ export const components = {
   'badge__background-color--warn': themeColors.warn,
   'badge__font-color--warn': themeColors.base,
   'badge__border-radius': radius.pill,
+  // choice + choicelist
+  'choice__background-color': themeColors.background,
+  'choice__background-color--checked': themeColors.primary,
+  'choice__background-color--disabled': themeColors['gray-lighter'],
+  'choice__background-color--inverse': 'transparent',
+  'choice__background-color--disabled--inverse': hexOpacity(themeColors['muted-inverse'], 15),
+  'choice__background-color--unchecked': themeColors.primary,
```
Comment (zarahzachz) [inline packages/design-system-tokens/src/themes/core/defaultTheme.ts:144]: Should `--unchecked` be the same color as `--checked`?

File: packages/design-system-tokens/src/themes/core/defaultTheme.ts [hunk 2]
```diff
@@ -131,6 +135,29 @@ export const components = {
...
+  'choice__border-color--error': themeColors.error,
+  'choice__border-color--inverse': themeColors.white,
+  'choice__border-color--left': themeColors.primary,
+  'choice__border-color--focus': themeColors['primary-darker'],
+  'choice__border-color--disabled--inverse': themeColors['gray-light'],
+  'choice__border-radius': '0px',
+  'choice__border-width': '2px',
+  'choice__color--unchecked': themeColors.white,
+  'choice__color--disabled': themeColors.muted,
+  choice__size: spacer['4'],
```
Comment (zarahzachz) [inline packages/design-system-tokens/src/themes/core/defaultTheme.ts:157]: Does this key not being in quotes impact anything?

File: packages/design-system-tokens/src/themes/healthcare/defaultTheme.ts [hunk 3]
```diff
@@ -135,6 +135,29 @@ const components = {
   'badge__background-color--warn': themeColors.warn,
   'badge__font-color--warn': themeColors.base,
   'badge__border-radius': radius.pill,
+  // choice + choicelist
+  'choice__background-color': themeColors.background,
+  'choice__background-color--checked': themeColors.primary,
+  'choice__background-color--disabled': themeColors['gray-lighter'],
+  'choice__background-color--inverse': 'transparent',
+  'choice__background-color--disabled--inverse': hexOpacity(themeColors['muted-inverse'], 15),
+  'choice__background-color--unchecked': themeColors.primary,
```
Comment (zarahzachz) [inline packages/design-system-tokens/src/themes/healthcare/defaultTheme.ts:144]: Same question as above - curious that `--checked` and `--unchecked` receive the same value

File: packages/design-system-tokens/src/themes/healthcare/defaultTheme.ts [hunk 4]
```diff
@@ -135,6 +135,29 @@ const components = {
...
+  'choice__border-color--error': themeColors.error,
+  'choice__border-color--inverse': themeColors.white,
+  'choice__border-color--left': themeColors.primary,
+  'choice__border-color--focus': themeColors['primary-darker'],
+  'choice__border-color--disabled--inverse': themeColors['gray-light'],
+  'choice__border-radius': '0px',
+  'choice__border-width': '2px',
+  'choice__color--unchecked': themeColors.white,
+  'choice__color--disabled': themeColors.muted,
+  choice__size: spacer['4'],
```
Comment (zarahzachz) [inline packages/design-system-tokens/src/themes/healthcare/defaultTheme.ts:157]: No quotes?

File: packages/design-system-tokens/src/themes/medicare/defaultTheme.ts [hunk 5]
```diff
@@ -169,6 +169,29 @@ const components = {
   'badge__background-color--warn': themeColors.warn,
   'badge__font-color--warn': themeColors.base,
   'badge__border-radius': radius.pill,
+  // choice + choicelist
+  'choice__background-color': themeColors.background,
+  'choice__background-color--checked': themeColors['primary-alt'],
+  'choice__background-color--disabled': themeColors['gray-lighter'],
+  'choice__background-color--inverse': 'transparent',
+  'choice__background-color--disabled--inverse': hexOpacity(themeColors['muted-inverse'], 15),
+  'choice__background-color--unchecked': themeColors.primary,
```
Comment (zarahzachz) [inline packages/design-system-tokens/src/themes/medicare/defaultTheme.ts:178]: See above.

File: packages/design-system-tokens/src/themes/medicare/defaultTheme.ts [hunk 6]
```diff
@@ -169,6 +169,29 @@ const components = {
...
+  'choice__border-color--error': themeColors.error,
+  'choice__border-color--inverse': themeColors.white,
+  'choice__border-color--left': themeColors.primary,
+  'choice__border-color--focus': themeColors['primary-darker'],
+  'choice__border-color--disabled--inverse': color['granite-300'],
+  'choice__border-radius': radius.default,
+  'choice__border-width': '2px',
+  'choice__color--unchecked': themeColors.white,
+  'choice__color--disabled': themeColors.muted,
+  choice__size: spacer['3'],
```
Comment (zarahzachz) [inline packages/design-system-tokens/src/themes/medicare/defaultTheme.ts:191]: See above.

File: packages/design-system/src/styles/components/_Choice.scss [hunk 7]
```diff
@@ -58,23 +56,23 @@ label,
...
   &::before {
-    box-shadow: inset 0 0 4em 1em $choice-checked-background-color;
+    box-shadow: inset 0 0 4em 1em $choice__background-color--checked;
     content: '';
     grid-area: 1/-1;
-    height: $choice-size;
+    height: $choice__size;
     opacity: 0;
     // transition: 120ms opacity ease-in-out;
     width: $choice-size;
```
Comment (zarahzachz) [inline packages/design-system/src/styles/components/_Choice.scss:78]: Should `width` receive token?

File: packages/design-system-tokens/src/themes/core/defaultTheme.ts (hunk 2 above)
Comment (forrestbaer) [reply packages/design-system-tokens/src/themes/core/defaultTheme.ts:157]: Good point, no it doesn't, more readable that way, updated.  I just got used to needing it with color vars.

File: packages/design-system-tokens/src/themes/healthcare/defaultTheme.ts (hunk 3 above)
Comment (forrestbaer) [reply packages/design-system-tokens/src/themes/healthcare/defaultTheme.ts:144]: Actually these unchecked variables are unused, will remove, was a part of the audit that was identified as an opportunity but we can always add them if needed.

File: packages/design-system/src/styles/components/_Choice.scss (hunk 7 above)
Comment (forrestbaer) [reply packages/design-system/src/styles/components/_Choice.scss:78]: hmm, definitely, thanks.

**Review by zarahzachz — APPROVED** [review]

**Pull request merged**